pkg/client/clientset/versioned/typed/crd/v1alpha1/clusternetworkpolicy.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2021 Antrea Authors
+// Copyright 2023 Antrea Authors
```

Comment:
Why is this changed?

Reply:
I reformated this file accidentally, resulting in the import files being sorted.  Then `make code-gen` command will edit this file every time even if I undo my format operation. Maybe I could edit the year manually.

Reply:
Not sure what's kind of reformat are you referring to, but I feel it's better to avoid this change if there is no code change on the CRD.